### PR TITLE
Sept 2017

### PR DESCRIPTION
**Pull request by seanwestfall**

### CODE REVIEW

File: client/cards/next.js [hunk 1]
```diff
@@ -18,6 +13,11 @@ module.exports = h('.next',
   */
 
   h('div', {style: {'margin-top': '10%'}},
+    h('h2', 'Justin Dorfman'),
+    h('h2', 'Let’s talk about SRI')
+  ),
```
Comment (ashwell) [inline client/cards/next.js:18]: I believe @jcblw said we have a second speaker lined up but I don't see it on the spread sheet.

@jcblw can we get this info to put up on the slides tonight?

File: client/cards/speaker1.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 var h = require('hyperscript')
 
 module.exports = h('.speaker',
-  h('h1', 'Why you should (almost) always use stateful components in React'),
-  h('h2', 'Aaron Craig'),
+  h('h1', 'Peer to Peer Video Streaming with WebRTC'),
+  h('h2', 'Brian Mau'),
 
-  h('img', {src: '/img/speakers/aaron-craig.jpeg', style: {width: '25%'}})
+  h('img', {src: '/img/speakers/brian-mau.jpeg', style: {width: '25%'}})
```
Comment (ashwell) [inline client/cards/speaker1.js]: I think you can safely bump this width up to `50%`

**Review by ashwell — CHANGES_REQUESTED** [review]
Looks good, I think we can make Brian's picture a little bigger.

Also wondering if @jcblw has a speaker locked down that we can add to the next event card.

**Comment by seanwestfall** [comment]
@ashwell Requested changes made. Though don't have the info @jcblw has for the second October speaker.

**Review by ashwell — APPROVED** [review]

**Pull request merged**